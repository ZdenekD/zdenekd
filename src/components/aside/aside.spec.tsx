import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Aside from '.';

describe('Aside', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Aside />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-aside');

        expect(component.exists()).toBe(true);
    });
});
