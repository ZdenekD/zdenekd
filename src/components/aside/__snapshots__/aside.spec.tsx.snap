// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside match snapshot 1`] = `
<aside class="aside"
       data-test="component-aside"
       style="transform:translateY(-100%) translateZ(0)"
>
  <nav class="nav"
       data-test="component-nav"
  >
    <ul class="list"
        style="opacity:0"
    >
      <li class="item"
          data-test="component-item"
          style="opacity:0;transform:translateY(-6vh) translateZ(0)"
      >
        <a href="/"
           class="anchor link active"
           data-test="component-anchor"
        >
          Úvodní stránka
        </a>
      </li>
      <li class="item"
          data-test="component-item"
          style="opacity:0;transform:translateY(-6vh) translateZ(0)"
      >
        <a href="/projects"
           class="anchor link "
           data-test="component-anchor"
        >
          Projekty
        </a>
      </li>
      <li class="item"
          data-test="component-item"
          style="opacity:0;transform:translateY(-6vh) translateZ(0)"
      >
        <a href="/contact"
           class="anchor link "
           data-test="component-anchor"
        >
          Kontakt
        </a>
      </li>
    </ul>
  </nav>
</aside>
`;
